providers/capa/standard/capa_test.go

```diff
@@ -1,11 +0,0 @@
-package standard
```

Comment:
I think keeping these files around would be best as this is where the provider-specific tests would eventually go.

Reply:
Done